### ISSUE TITLE
[REQUEST] Incorporate Chrome's tagged PDF export feature

### ISSUE THREAD

**Issue by corey-slade** [alixaxel/chrome-aws-lambda#229]
## What would you like to have implemented?

In Chrome 85 an update was implemented to allow tagged PDFs to be generated, carrying over the alt text from the HTML to the PDF it creates. (https://blog.chromium.org/2020/07/using-chrome-to-generate-more.html#:~:text=Using%20Chrome%20to%20generate%20more%20accessible%20PDFs,-Wednesday%2C%20July%2029&text=Starting%20in%20Chrome%2085%20(scheduled,%22Save%20as%20PDF%22%20destination.)
For whatever reason when I generate a PDF from an HTML file using this package the tags are not carried over, whereas if I manually print to pdf from the same HTML file they are.

## Why would it be useful?

It would enable us to easily meet Accessibility requirements

**Comment by pgangloff-hbs** [comment]
we wonder why this was never merged into the main branch. This would improve PDF accessibility so much (upon generating/printing it to PDF in Chrome). For now, we'll add a "print" icon on our web pages that makes a backend call to create the PDF (and programatically set the title, alt text, etc). But even then, nothing would stop a user from hitting CTRL+P (or CMD+P) and end up with a bad PDF, created by Chrome, that is not very accessibly to vision impaired. This should be a "critical", not a nice to have in this day of digital accessibility... Thanks!
